pluto/src/vis/schematic/registry.ts

```diff
@@ -720,7 +720,7 @@ const switch_: Spec<SwitchProps> = {
   name: "Switch",
   key: "switch",
   Symbol: Switch,
-  Form: CommonToggleForm,
+  Form: () => CommonToggleForm({ hideInnerOrientation: true }),
```

Comment:
You're rendering a react component directly as a function. Move this into a new component within Forms.tsx